src/main/java/marquez/db/DbTableInfoDao.java

```diff
@@ -0,0 +1,27 @@
+package marquez.db;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import marquez.common.models.Db;
+import marquez.common.models.Schema;
+import marquez.db.mappers.DbTableInfoRowMapper;
+import marquez.db.models.DbTableInfoRow;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.BindBean;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+@RegisterRowMapper(DbTableInfoRowMapper.class)
+public interface DbTableInfoDao {
+  @SqlUpdate(
+      "INSERT INTO db_table_info (uuid, db, table) " + "VALUES (:uuid, :db.value, :schema.value)")
```

Comment:
supppperrrrr nit: this SQL fits on one line now, so need for the `+`

Reply:
also the `schema.value` is going into the `table` column?

Reply:
need to add `ON CONFLICT DO NOTHING`

Reply:
this table will also need a unique index on `(db, schema)`, or how do we avoid this table getting a new row every time we insert a new `db_table_version` even when the `db_table_info` does not change?